action.yml

```diff
@@ -18,6 +18,10 @@ inputs:
     description: 'Fail the action in case we found at least this vulnerability risk'
     required: false
     default: none
+  type:
```

Comment:
nit: should we call this scan_type as well to match ptt ?

Reply:
We can, but it seems simpler this way. `ptt` originally was closer to the API, and the API names is `scan_type`